packages/http-client-java/eng/scripts/Initialize-Repository.ps1

```diff
@@ -68,6 +68,16 @@ try {
     # Write-Host "Updated PATH: $env:PATH"
     # Invoke-LoggedCommand "java -version"
     # Invoke-LoggedCommand "mvn -version"
+    
+    # install npm packages at repository root
+    Push-Location "$packageRoot/../../"
+    try {
+        Invoke-LoggedCommand "npm install -g pnpm" -GroupOutput
+        Invoke-LoggedCommand "pnpm install" -GroupOutput
+    }
+    finally {
+        Pop-Location
+    }
```

Comment:
We do it here (and `npm run lint`) for a reason that eslint at root is not able to check lib that not installed (here the TCGC lib).

The alternative would be just remove the changes to emitter scripts, and solely depends on the Lint action.
This could miss some deprecation from TCGC. But when dev runs `npm run lint` in local, they should still see the error. And I think this would be good enough.